data/repositories.toml

```diff
@@ -30,4 +30,5 @@ repositories = [
   'github.com/crossplaneio/crossplane',
   'github.com/kubernetes/website',
   'github.com/darrenburns/ward'
```

Comment:
Oops, looks like you forgot a comma here. But nevermind, that happens a lot with all of us.